app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -195,13 +213,8 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['miqService', 'API
   };
 
   var getStorageManagers = function(data) {
-    vm.storageManagers = data.resources;
-
-    // If storage manager ID was provided, we need to refresh the form and show
-    // corresponding form fields.
-    if (storageManagerId) {
-      vm.storageManagerChanged(vm.cloudVolumeModel.storage_manager_id);
-    }
+    // Can handle list of all managers or a single manager.
+    vm.storageManagers = data.resources ? data.resources : [data];
```

Comment:
@himdel this is where we store value "as is" - and it's integer. If I convert storageManagerId into a string (in the haml) then nothing gets selected in the drop-down. Is it possible that API returns integers? Or you think controller somehow parses it wrong?

I say we do separate PR for this as it's not related to this BZ. Will check it a bit later when I checkout fine branch again.

Reply:
The API will most likely return compressed ids, in the format like `"1r23"`. You can use `miqUncompressedId` to convert to `'1000000000023'` in JS, or you can use `to_cid` ruby-side to convert *to* that format.